### PR TITLE
Internal: Convert Gap control for kits and templates [ED-11782]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: assets/dev/js/editor/controls/gaps.js [hunk 1]
```diff
@@ -41,10 +41,6 @@ ControlGapItemView = ControlDimensionsView.extend( {
 
 		this.setSettingsModel( values );
 	},
-
-	getControlPlaceholder() {
-		return '';
-	},
```
Comment (davseve) [inline assets/dev/js/editor/controls/gaps.js:47]: Show gaps control placeholders

File: includes/controls/gaps.php [hunk 2]
```diff
@@ -60,4 +59,74 @@ protected function get_dimensions() {
...
+						</button>
+					</li>
+				</ul>
+			</div>
+		</div>
+		<# if ( data.description ) { #>
+			<div class="elementor-control-field-description">{{{ data.description }}}</div>
+		<# } #>
+		<?php
+	}
```
Comment (davseve) [inline includes/controls/gaps.php:131]: Related to the placeholder appearence

File: includes/controls/gaps.php [hunk 3]
```diff
@@ -60,4 +59,74 @@ protected function get_dimensions() {
 			'row' => esc_html__( 'Row', 'elementor' ),
 		];
 	}
+
```
Comment (TzviRabinovitch) [inline includes/controls/gaps.php:62]: extract to different PR and create a story if needed

File: modules/control-replacer/module.php [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (TzviRabinovitch) [inline modules/control-replacer/module.php:1]: refactor to a single-responsibility command  pattern

suggestions :
modules/control-converters/flex-container-gaps-converter.php

with an execute function that does the conversion

**Review by TzviRabinovitch — CHANGES_REQUESTED** [review]

File: modules/control-replacer/module.php (hunk 4 above)
Comment (louiswol94) [reply modules/control-replacer/module.php]: Het @TzviRabinovitch I think this is overkill. `on_import()` is already like a command that is being executed, it does not really make sense to then execute another command. The logic can just be inside `on_import()` directly. But because the same logic is needed in the Upgrade scripts for example we need to either make it in a trait or just a static function inside the container.php file. It's basically just a small piece of logic that needs to be shared.

**Review by TzviRabinovitch — APPROVED** [review]
